CNAME

```diff
@@ -1 +1 @@
-a11yproject.com
```

Comment:
This is a mistake I think? We need the domain name in the CNAME so it resolves properly on the Github side of things.